src/cldfviz/multiparameter.py

```diff
@@ -102,6 +103,17 @@ def __init__(self, ds, pids, include_missing=False, glottolog=None, language_pro
             for row in ds.iter_rows('CodeTable', 'id', 'parameterReference', 'name'):
                 if row['parameterReference'] in self.parameters:
                     codes[row['parameterReference']][row['id']] = row['name']
+        # sort codes for boolean values
```

Comment:
I am sorry, I forgot to push this, and my suggestion is maybe naive and not good, but it is important to account for a proper sorting for the True-False-None-Values, since otherwise, sorting depends on write order in CSV-files, which is not reliable, and leads to mix-up of True-False-plots in our lexibank study (and not only there). So users must have some way to dictate the order of values.

Reply:
Ok. I would have thought that "write-order in codes.csv" is exactly the way to control color assignment, but I realize that there may be all sorts of occasions, where this is not sufficient. Will think a bit about a solution.

Reply:
I'd be leaning towards allowing users to pass in explicit color maps, either from JSON files or as JSON strings, à la
```
--colormaps '{"True": "#abc", "False": "#000"}'
```
If we read this into `OrderedDict`s, we can also specify code order that way.

Reply:
@LinguList what about:
```
$ cldfbench cldfviz.map --parameters ArmAndHand --colormaps '{"True":"#a00","False":"#00a","None":"#000"}' --markersize 15 --missing-value "#fff" ../../lexibank/lexibank-analysed/cldf/lexicon-metadata.json 
```

Reply:
Sorry, did not see this before, but this is a perfectly fine solution!

Reply:
I like the solution also because it is not easy to control what users will come up with, and the typical case where order is important is the case of three values, etc. not more. This means, we can also get rid of the two `lb-X` colormaps, I proposed, and just pass our own values.